src/interface/src/app/types/plan.types.ts

```diff
@@ -0,0 +1,17 @@
+import { Region } from "./region.types";
+
+export interface Plan extends BasePlan {
+  id: string;
+  name: string;
+  ownerId: string;
+  region: Region;
+  planningArea: GeoJSON.GeoJSON;
+}
+
+export interface BasePlan {
```

Comment:
what's the difference between BasePlan and Plan?

Reply:
BasePlan's id is optional - so in the case of creating a new plan, it doesn't have an id before it's created in the database.